src/core/configuration/DefaultConfig.ts

```
@@ -490,34 +500,28 @@
+    const defenderType = defenderIsPlayer ? defender.type() : null;
+
+    if (defenderIsPlayer) {
       for (const dp of gm.nearbyUnits(
         tileToConquer,
         gm.config().defensePostRange(),
         UnitType.DefensePost,
       )) {
-        if (dp.unit.owner() === defender) {
-          mag *= this.defensePostDefenseBonus();
-          speed *= this.defensePostDefenseBonus();
+        if (dp.unit.owner() == defender) {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)